codegen/mustache/java/pojo.mustache

```diff
@@ -0,0 +1,414 @@
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+import com.google.gson.TypeAdapterFactory;
+import com.google.gson.reflect.TypeToken;
+
+import java.lang.reflect.Type;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+
+import com.whispir.client.ApiResponse;
+import {{invokerPackage}}.JSON;
+
+/**
+ * {{description}}{{^description}}{{classname}}{{/description}}
+ */
+
+public class {{classname}} {
+  @SerializedName("docId")
+  private String docId;
+  @SerializedName("rawOutput")
+  private String rawOutput;
+  // Additional getters
+  public String getDocId() {
```

Comment:
Could we replace all text instances of `docId` here with just `id`? I think this is possible now given that we've removed all `id` attributes from all models. This also aligns with Stripe API.

Reply:
Hmm, on second thought, removal of the `id` attribute from the response model of each `GET /<resource>` endpoint actually makes it impossible to use the SDK to perform a `GET /<resource>` query then retrieve the full model for the resource using the `id` param.

We may need to revert https://github.com/whispir/openapi/pull/182/files#diff-d910ba2ef878f7db0223a966b81c8b3f3b65027bb39e4431bb05140171eece39L9223. Let's discuss.